### ISSUE TITLE
Github action resource issue

### ISSUE THREAD

**Issue by yiheng-wang-nv** [Project-MONAI/model-zoo#496]
```
[pipenv.exceptions.InstallError]: ERROR: Could not install packages due to an OSError: [Errno 28] No space left on device
```

reference: 
https://github.com/Project-MONAI/model-zoo/actions/runs/6010712328/job/16304196611?pr=479

need to figure out how to reduce the pipenv cache usage.

**Comment by yiheng-wang-nv** [comment]
also impact this PR:
https://github.com/Project-MONAI/model-zoo/actions/runs/6021275704/job/16333877046?pr=483
cc @KumoLiu

**Comment by yiheng-wang-nv** [comment]
this issue is solved in #483 via adding `pipenv --clear`
